Alignment/APEEstimation/test/cfgTemplateDesign/createStep1.bash

```diff
@@ -42,12 +42,12 @@ export datasetName
 ## number of input files
 export nFiles
 ## Input file base
-cafDir="\/store\/caf\/user\/cschomak\/wlnu"
+cafDir="\/store\/caf\/user\/cschomak\/Skims\/MC\/PhaseI\/DY"
```

Comment:
We are 3 users that use skims created by one user. Thus, we need to specify the user name